BridgeAppSDK/SBAUser.swift

```diff
@@ -157,6 +158,24 @@ public final class SBAUser: NSObject, SBAUserWrapper {
         }
     }
     
+    public var profileImage: UIImage? {
+        get {
+            if let profileImageData = getKeychainObject(kProfileImagePropertyKey) as? Data {
+                return UIImage(data: profileImageData)
+            } else {
+                return nil
+            }
+        }
+        set (newValue) {
+            if let newValueUnwrapped = newValue {
+                let dataValue = UIImageJPEGRepresentation(newValueUnwrapped, 1.0)
```

Comment:
I went with UIImageJPEGRepresentation because it is faster and less storage space than lossless UIImagePNGRepresentation.  We could switch the property to be Data? to provide the developer more control over the storage format, but I think this provides an easier interface.